tests/arp/test_arpall.py

```diff
@@ -11,6 +11,17 @@
     pytest.mark.topology('t1')
 ]
 
+logger = logging.getLogger(__name__)
+
+
+def collect_info(duthost):
```

Comment:
Do you want to dump this info into a file? This is called at multiple places and will make the log file too long and difficult to comprehend.

Reply:
My understanding is that this is a debug change to catch the mac change issue. 

Xin, are we going to revert this change once the issue is confirmed?

Reply:
Maybe make it optional and enable it by test option so that we can enable it only for Mellanox platform?

Reply:
This is a temporary change. It will be reverted once the issue is reproduced. 
This change will add extra logs to the debug level log file. IMO, it is an acceptable overhead. Anyway, I'll update it to run just for Mellanox platform.